pyproject.toml

```diff
@@ -20,5 +20,5 @@ max-line-length = 120
 
 [tool.vulture]
 min_confidence = 90
-exclude = ["/tests/"]
+exclude = ["/tests/", "/.venv/"]
```

Comment:
This was actually an issue I was experiencing when running linters, and vulture was taking forever parsing my .venv's

Reply:
That depends entirely on how you run it.

I've just tested this with pre-commit without the exclude option and it worked fine. Also doesn't seem to be an issue for the CI

Reply:
As discussed offline: vulture does not deal elegantly with the files it receives from pre-commit on all platforms for some reason. For the time being, let's add `"*venv*"` as a "temporary" solution.

Reply:
The vulture hook uses `pass_filenames: false` so doesn't get the list of files from pre-commit. The other hooks don't use that option so don't have this problem. `"*venv*"` might accidentally match too much, I've added both `"/venv/"` and `"/.venv/"` now.

Reply:
Ah good catch. I'm fine with this. Although I would prefer to have `pass_filenames` enabled and add the venv patterns to the root exclude. This would make the rest of the config files a bit cleaner since we don't have to repeatedly add these patterns.